components/automate-ui/src/app/pages/api-token/details/api-token-details.component.html

```diff
@@ -20,29 +20,42 @@
           <tr class="detail-row">
             <td class="header-column">{{ token?.id }}</td>
             <td class="header-column">{{ token?.created_at | datetime:'ddd, MMMM D, YYYY [at] H:mm a' }}</td>
-            <td class="header-column">{{ status }}</td>
+            <td class="header-column">{{ status | capitalize }}</td>
           </tr>
         </tbody>
       </table>
-        <chef-tab-selector [value]="tabValue">
-          <chef-option value='details'>Details</chef-option>
-        </chef-tab-selector>
-      </chef-page-header>
+      <chef-tab-selector [value]="tabValue">
+        <chef-option value='details'>Details</chef-option>
+      </chef-tab-selector>
+    </chef-page-header>
     <section class="page-body">
-      <form [formGroup]="updateNameForm">
+      <form [formGroup]="updateForm">
         <chef-form-field>
-          <label>Name *</label>
-          <input chefInput formControlName="name" type="text" (keyup)="handleNameChange()">
+        <!-- Once the changes from https://github.com/chef/automate/pull/1115 merge,
+            uncomment this and get rid of `handleChange`:
+        <chef-form-field [watch]="nameCtrl.value" [control]="nameCtrl"> -->
+          <label>Name <span aria-hidden="true">*</span></label>
+          <input chefInput formControlName="name" type="text" (keyup)="handleChange()">
+          <chef-error *ngIf="(nameCtrl.hasError('required') || nameCtrl.hasError('pattern')) && nameCtrl.dirty">
+            Name is required.
+          </chef-error>
+        </chef-form-field>
+        <chef-form-field>
+          <label>Status <span aria-hidden="true">*</span></label>
+          <chef-radio ngDefaultControl formControlName="status">
```

Comment:
This is something we should fix on the component, but wanted to note that I can't tab through the chef-radio options. Tabbing just jumps from the name input to the save button.